File: scripts/algorithms/P/Percentage of Letter in String/Percentage of Letter in String.cpp
class Solution {
public:
    int percentageLetter(string s, char letter) {
        int count=0;
        for(int i=0;i<s.length();i++)
        {  if(s[i]==letter)
            {
                count++;
            }
        }
        return (count*100)/s.length();
        
    }
};

File: scripts/algorithms/P/Percentage of Letter in String/Percentage of Letter in String.java
// Runtime: 0 ms (Top 100.0%) | Memory: 40.55 MB (Top 34.5%)

class Solution {
    public int percentageLetter(String str, char letter) {
        int count=0;
        int n=str.length();
        for(int i=0;i<n;i++){
            if(str.charAt(i)==letter){

                count ++;
            }
        }
                 int per= (100*count)/n;
         return per;   
        
    }
}